### ISSUE TITLE
Error in Hypermorph

### ISSUE THREAD

**Issue by Paluck1Arora2** [voxelmorph/voxelmorph#498]
I have executed this code "https://ahoopes.github.io/hypermorph/tutorial.html" but getting this error at the time of training. Can anyone please resolve it.
Node 'training/Adam/gradients/gradients/vxm_dense_transformer/map/while_grad/vxm_dense_transformer/map/while_grad': Connecting to invalid output 154 of source node vxm_dense_transformer/map/while which has 154 outputs. Try using tf.compat.v1.experimental.output_all_intermediates(True).

**Comment by Paluck1Arora2** [comment]
Hi, 
Can anyone knows how to plot moving image 
# extract the first frame (i.e. the "moving" image) from unet input tensor
moving_image = tf.expand_dims(unet.input[..., 0], axis=-1)
ne.plot.slices(moving_image) # getting error at this place.
This is the link of voxelmorph tutorial https://colab.research.google.com/drive/1R7OCnA0loDHE0RADMrbS6Cg1A1UPt2UK?usp=sharing

**Comment by Paluck1Arora2** [comment]
> I have executed this code "https://ahoopes.github.io/hypermorph/tutorial.html" but getting this error at the time of training. Can anyone please resolve it. Node 'training/Adam/gradients/gradients/vxm_dense_transformer/map/while_grad/vxm_dense_transformer/map/while_grad': Connecting to invalid output 154 of source node vxm_dense_transformer/map/while which has 154 outputs. Try using tf.compat.v1.experimental.output_all_intermediates(True).

I'm getting the solution of this question "https://colab.research.google.com/drive/1UqRkpmz91pZbpX_zrlP3NC_3856urQJd?usp=sharing#scrollTo=vPDh0Nfvqwvv"

**Comment by ahoopes** [comment]
Does setting `tf.compat.v1.experimental.output_all_intermediates(True)` after importing tensorflow fix this?

I'm not sure I follow the plotting question.
